packages/ui-kit/src/components/shared.types.ts

```diff
@@ -7,38 +8,38 @@ import type { ThemeStateProps } from './ThemeProvider/ThemeProvider.types'
 
 export interface FallbackComponents {
   /**
-   * A fallback react component that will be used when the component is in loading state.
+   * A fallback react component that will be used when the component is in error state.
    *
-   * You can also use a callback to get the `LoaderProps` for your custom component or set a wrapper
-   * around the UI-Kit's Loader component.
+   * You can also use a callback to get the `ErrorFallbackProps` for your custom component or set a wrapper
+   * around the UI-Kit's ErrorFallback component.
    *
    * This component will be used by all the ThemeProvider's child components.
    */
-  loaderFallback?: ({
-    loaderProps,
-    Loader,
+  errorFallback?: ({
+    errorFallbackProps,
+    ErrorFallback,
```

Comment:
This and the change to loader, etc., is a breaking change, no? Or did we not release it yet?